tests/test_errors.py

```diff
@@ -0,0 +1,59 @@
+"""
+Tests in this file shall test functionlity
+not relying on access to vibium-cloud API
+"""
+
+import pytest
+import requests
+from mvg.exceptions import MVGAPIError, raise_for_status
+
+
+class MockResponse:
+    def __init__(self, status_code, reason, json_body):
+        self.status_code = status_code
+        self.reason = reason
+        self.json_body = json_body
+
+    def json(self):
+        return self.json_body
+
+    def raise_for_status(self):
+        if 400 <= self.status_code < 600:
+            raise requests.HTTPError()
+
+
+def mock_fail_request(json_body):
+    return MockResponse(400, "Bad Request", json_body)
+
+
+def assert_fail_response(msg, detail="No error message"):
+    assert msg == f"400 - Bad Request: {detail}"
+
```

Comment:
How are these two functions used? Do not seem to be invoked by other functions, and won't be called as a test function by pytest either?

Reply:
These were just something I was playing around with but decided was overkill. Seems I forgot to remove the functions